docs/pages/web/tagdata.js

```diff
@@ -0,0 +1,247 @@
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`TagData is available in 3 fixed sizes.
+          - **Large** has height of 48px. Text has a fixed size of 16px. 
+          - **Medium** has height of 40px. Text has a fixed size of 14px. 
+          - **Small** has height of 32px. Text has a fixed size of 14px.
+          `}
+          title="Size"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={sizes} name="Size Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="TagData should use a different fill color than the color of the surrounding UI. When used on a white background, it is recommend using a `baseColor` of white when on a gray UI background."
+          title="Base Color"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={baseColor} name="Base Color Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="TagData is available in the colors from our Data Visualization [Color Palette](https://gestalt.pinterest.systems/foundations/data_visualization/palette#12-Color-categorical-palette). Match each TagData's `color` to its respective data line, which will be visible when TagData is selected."
+          title="Colors"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={colors} name="Colors Variant" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description={`Use TagData's \`showCheckbox\` prop to display a checkbox. This is useful when presenting the user with a multi-select experience.`}
+          title="Group"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={group} name="Checkbox Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description={`If not disabled, Tags are dismissible by the "X" affordance, which triggers the \`onRemove\` callback. \`onRemove\` should be used to update the external state that is keeping track of TagDatas. If your app uses DefaultLabelProvider, a default accessibility label for the remove icon will be used. This can be overridden with a more specific label if desired.`}
+          title="Dismissible"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={dismissible} name="Dismissible Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="Use `tooltip` to display clarifying information on hover/focus. We recommend using tooltips to provide the user additional context/details."
```

Comment:
Consider not using a slash for readability in "…the user additional context or details."